src/main/kotlin/com/openlattice/shuttle/control/FlightPlanParameters.kt

```diff
@@ -0,0 +1,55 @@
+package com.openlattice.shuttle.control
+
+import com.dataloom.mappers.ObjectMappers
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import com.openlattice.mapstores.TestDataFactory
+import com.openlattice.shuttle.Flight
+import java.net.URL
+import java.util.*
+
+/**
+ * Represents the parameters required to create a flight plan (i.e. a Map<Flight, Payload>)
+ *
+ * @param sql the sql query to be used to pull cleaned data from postgres
+ * @param source postgres data source for pulling clean data
+ * @param sourcePrimaryKeyColumns the columns that are primary keys in the cleaned data
+ * @param flightFilePath the path to the flight yaml. If you would like the flight to
+ * be reloadable from this path, please store flight yaml on github and provide the path
+ * to it (i.e. https://raw.githubusercontent.com/pathToFlight.yaml)
+ * @param flight Flight object
+ */
+
+data class FlightPlanParameters(
+        @JsonProperty(SerializationConstants.SQL) var sql: String,
+        @JsonProperty(SerializationConstants.SRC) var source: Properties,
```

Comment:
Maaaaybe its just me but I would definitely prefer storing this as an `ImmutableMap` and just converting to `Properties` in `IntegrationService` since `Properties` is kind of a gross, fat class that extends a gross rage-inducing class `HashTable<Object, Object>` b/c they both come from `Java 1.0`